examples/pytorch-resnet/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "pytorch-resnet"
```

Comment:
Can we use example- as a prefix here to be consistent with the other test ?

Reply:
Yes!